wpimath/src/main/native/include/frc/estimator/DifferentialDrivePoseEstimator.h

```diff
@@ -214,8 +214,7 @@ class WPILIB_DLLEXPORT DifferentialDrivePoseEstimator {
 
  private:
   UnscentedKalmanFilter<5, 3, 3> m_observer;
-  KalmanFilterLatencyCompensator<5, 3, 3, UnscentedKalmanFilter<5, 3, 3>>
-      m_latencyCompensator;
+  TimeInterpolatableBuffer<Pose2d> m_poseBuffer{1.5_s};
```

Comment:
What's the derivation of the 1.5 seconds?

Reply:
The latency compensator uses 300 samples internally, so if you assume a 5ms sample period, that gives a backlog of 1.5 seconds. I think using a constant backlog duration is better than scaling it based on the sample period the user supplies.